Mage.Tests/src/test/java/org/mage/test/cards/single/dsk/UnableToScreamTest.java

```diff
@@ -0,0 +1,125 @@
+package org.mage.test.cards.single.dsk;
+
+import mage.abilities.mana.GreenManaAbility;
+import mage.constants.*;
+import org.junit.Test;
+import org.mage.test.serverside.base.CardTestPlayerBase;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * @author markort147
+ */
+public class UnableToScreamTest extends CardTestPlayerBase {
+
+    public static final String UNABLE_TO_SCREAM = "Unable to Scream";
+
+    @Test
+    public void cannotTurnFaceUpTest() {
+        String akroma = "Akroma, Angel of Fury";
+        String breakOpen = "Break Open";
+
+        addCard(Zone.HAND, playerB, akroma);
+        addCard(Zone.BATTLEFIELD, playerB, "Mountain", 6);
+
+        addCard(Zone.HAND, playerA, UNABLE_TO_SCREAM);
+        addCard(Zone.HAND, playerA, breakOpen, 1);
+        addCard(Zone.BATTLEFIELD, playerA, "Island", 2);
+        addCard(Zone.BATTLEFIELD, playerA, "Mountain", 1);
+
+        castSpell(2, PhaseStep.PRECOMBAT_MAIN, playerB, akroma + " using Morph");
+        castSpell(3, PhaseStep.PRECOMBAT_MAIN, playerA, UNABLE_TO_SCREAM, EmptyNames.FACE_DOWN_CREATURE.getTestCommand(), true);
+        castSpell(3, PhaseStep.PRECOMBAT_MAIN, playerA, breakOpen, EmptyNames.FACE_DOWN_CREATURE.getTestCommand());
+
+        setStopAt(4, PhaseStep.BEGIN_COMBAT);
```

Comment:
All new tests must run under strict mode (with full and correct choice commands instead AI decisions).

So insert `setStrictChooseMode(true);` before each execute/setStopAt call.